src/main/resources/listLookup.json

```diff
@@ -198,5 +198,15 @@
         "welshJurisdictions": ["Tribiwnlys"],
         "restrictedProvenances": [],
         "defaultSensitivity": ""
+    },
+    "OPA_PRESS_LIST": {
+        "friendlyName": "Online Plea and Allocation Press List",
+        "welshFriendlyName": "Rhestr y Wasg – Pledio Ar-Lein a Dyrannu (OPA)",
+        "shortenedFriendlyName": "OPA Press List",
+        "url": "opa-press-list",
+        "jurisdictions": ["Crown"],
```

Comment:
With the changes to PI_AAD on the list type, I just want to double check Crown is correct?

Reply:
Checked with Ogechi and changing it to "Magistrates"